stdlib/camlinternalQuote.ml

```diff
@@ -0,0 +1,2766 @@
+type 'a lam = 'a
```

Comment:
Missing MIT licence header - needed for all new files.  See `dwarf_inlined_frames.ml`